### DISCUSSION TITLE
MSP VTX OSD channel designator changes at arming

### DISCUSSION THREAD

**Discussion by MrMap** [iNavFlight/inav#9836] [Support]
## Steps to Reproduce

1. Start out with e.g. Raceband Ch5 Pwr 2, using the EDGETX ELRS VTX LUA, confirm OSD display correct CH: R5 : 2
2. Arm (Aux 1 / CH5)
3. Watch the OSD changes to CH: A1: 2
4. The VTX Channel has not changed, it will still be on R5

## Additional context
I use HDZero goggles, all FW  22082023
HDZ Freestyle v1, current FW _22082023

Here is a short DVR video demonstrating the phenomena.
https://youtu.be/1cz541mFGg8
I have considered if this maybe is normal but I think it is not?

---

- FC Board name and vendor:
- # version
# INAV/SPEEDYBEEF405WING 7.0.0 Sep 11 2023 / 12:56:24 (3a2412e5)
# GCC-10.3.1 20210824 (release)

**Comment by MrD-RC** [comment]
My friend had this. It’s an HDZero bug. Downgrading the HDZero firmware fixed the issue for him. For some reason arming changes the channel.

**Comment by MrMap** [comment]
> My friend had this. It’s an HDZero bug. Downgrading the HDZero firmware fixed the issue for him. For some reason arming changes the channel.

Aha, interesting, thank you. Though seems it does not actually change the channel in use, but it makes it look like it and this causes much confusion, like (airborne) - oh shit, did I screw up my channel selection - am I stepping on somebody now...?

So, do you know if your friend submitted this as an issue on the HDZero Github?

**Comment by MartinHugh** [comment]
> My friend had this. It’s an HDZero bug. Downgrading the HDZero firmware fixed the issue for him. For some reason arming changes the channel.

Interesting. Could you please verify which firmware and which version.  I am assuming your friend is referring to the VTX firmware, and to do MSP-VTX it would have needed v1.4.0.  If he has rolled back to anything earlier MSP-VTX is not supported anyway. 

Any other details on exactly what was changed would be helpful.

**Comment by MrD-RC** [comment]
All I know is that he tried the latest HDZero firmware (this would be around 3 or 4 weeks ago) and he had problems with the channel; as described here. He rolled back to the previous version, and it all worked fine. I don’t use HDZero myself. I’ll see if he knows the version number.

**Comment by aerostarlegacy** [comment]
Hello, I think I am also having this issue. I have goggles and race V2 vtx on newest firmware. When using stick commands in osd to change channel or power the OSD reflects the new values, but the channel and power do not actually change. @MrD-RC  were you able to get that extra info from your friend? it may help me or others.
Thanks!

**Comment by b14ckyy** [comment]
Will move this to discussion as it seems to be not a INAV issue.
